patches/tModLoader/Terraria/Program.cs.patch

```diff
@@ -130,3 +134,24 @@
  #endif
  			}
  			catch {
+ 			}
++		}
++
++		private static string GetTip(Exception e) {
```

Comment:
It might be preferred to not have this method in Program.cs, but one of our own files.
To reduce the patch, do what I suggested.